src/dso_api/settings.py

```diff
@@ -48,8 +48,9 @@
     "rest_framework",
     "rest_framework_gis",
     "gisserver",
+    "schematools.contrib.django",
```

Comment:
Heb je in `schematools/contrib/django/__init__.py` een `default_app_config` staan? Anders wordt de appname immers "django". Dat blokkeert deze PR niet, alleen iets om in schematools te fixen.

Reply:
Yup https://github.com/Amsterdam/schema-tools/blob/master/schematools/contrib/django/__init__.py
en yup: https://github.com/Amsterdam/schema-tools/blob/master/schematools/contrib/django/apps.py#L7